app/policies/university_policy.rb

```diff
@@ -1,8 +1,23 @@
 class UniversityPolicy < ApplicationPolicy
   class Scope < Scope
     # NOTE: Be explicit about which records you allow access to!
-    # def resolve
-    #   scope.all
-    # end
+    def resolve
+      scope.all
+    end
+  end
+
+  def show?
+    true
+  end
+
+  def create?
+    user.present?
+  end
+
+  def edit?  
```

Comment:
the 'edit?' the method can be removed. This is because university_policy inherits from application policy. Within application policy 'edit?' calls the update method so only the update method is needed in our app I believe.